app/components/modals/signature/SignatureModal.tsx

```
@@ -35,171 +35,159 @@ import { isDataUrl } from "@/lib/helpers";
 // Types
 import { SignatureTabs } from "@/types";
 
-type SignatureModalProps = {};
+type SignatureModalProps = Record<string, never>;
 
 const SignatureModal = ({}: SignatureModalProps) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)